object-store-client-play-26/src/main/scala/uk/gov/hmrc/objectstore/client/play/package.scala

```diff
@@ -63,5 +58,5 @@ package play {
   }
 
   // simplify by unifying the type classes required for the Play implementations
-  trait PlayMonad[F[_]] extends MonadError[F, PlayObjectStoreException] with MonadFuture[F]
+  trait PlayMonad[F[_]] extends MonadError[F, Exception] with MonadFuture[F]
```

Comment:
Given that we have lost the types of exceptions that can be thrown, should we add a `@throws` scaladoc to the methods of `ObjectStoreClient` to list them?

Reply:
Do we want the library users to be aware of these exceptions as they might try handling them? Usually, the bootstrap-play lib will handle these exceptions and turn them into the appropriate HTTP response.